### DISCUSSION TITLE
GameCube Controller Adapter support on vWii through Priiloader?

### DISCUSSION THREAD

**Discussion by ABOhiccups** [DacoTaco/priiloader#369] [Ideas]
Well it be possible to make GameCube Controller Adapter (for Wii U and Nintendo Switch) compatible with vWii by using a Priiloader Plugin?

**Comment by Ingunar** [comment]
It is not a thing Priiloader can do, but Priiloader can help with it.
You would need cIOS with [Fakemote](https://github.com/xerpi/fakemote)
Fakemote still has limited controller support, it is a work in progress. (Only Supports DS3 and DS4 currently)

You can use [Force Disc Games to run under IOS[248,249,250,251]](https://dacotaco.github.io/priiloader/docs/HACKSLIST.html#force-disc-games-to-run-under-ios248249250251) and if the cIOS supports the GameCube Controller Adapter, it will work.

**Reply by DacoTaco** [reply]
depends on what @ABOhiccups wants.
if they want to use it in priiloader then it would best be added to libogc
if they want it in games/wiiware etc then a ios patch is indeed best

**Reply by Ingunar** [reply]
I think that they meant for the general vWii use.
But about adding it to libogc, It is not useless to have it included.
@GaryOderNichts  was reverse engineering it [a couple of months ago](https://github.com/GaryOderNichts/hpad_test) but that only covers the WiiU homebrew side.
